content/terraform/v1.12.x/docs/language/block/moved.mdx

```diff
@@ -1,23 +1,27 @@
 ---
-page_title: moved block configuration reference
+page_title: moved block reference for the Terraform configuration language
 description: Learn about the `moved` block that you can specify in Terraform configurations. The `moved` block programmatically changes the location of a resource.
 ---
 
 # `moved` block reference
 
-This topic provides reference information for the `moved` block. Use this block to programmatically change the address of a resource. Refer to [Refactoring](/terraform/language/modules/develop/refactoring) for details about how to use the `moved` block in your Terraform configurations.
+This topic provides reference information for the `moved` block.
+
+## Introduction
+
+The `moved` block programmatically changes the address of a resource. Refer to [Refactoring](/terraform/language/modules/develop/refactoring) for details about how to use the `moved` block in your Terraform configurations.
 
 ## Configuration model
 
-The `moved` block supports the following arguments:
+The following list outlines field hierarchy, language-specific data types, and requirements in the `moved` block.
 
-- [`moved`](#moved): &nbsp block
-  - [`from`](#moved):  &nbsp reference | required
-  - [`to`](#moved):  &nbsp reference | required
+- [`moved`](#moved): map
```

Comment:
Just want to verify that this is indeed technically a map and not a block? Looking at the other docs I would expect this to be `block` but might be missing context.

Reply:
Nope, you're right. This page reverted to a different form somehow 🤨